action-allowedlist/approved.json

```diff
@@ -21,16 +21,28 @@
     },
     {
       "actionLink": "actions/checkout",
-      "actionVersion": "5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f"
+      "actionVersion": "5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f",
+      "tag": "v2.3.4"
     },
     {
       "actionLink": "actions/checkout",
       "actionVersion": "ec3a7ce113134d7a93b817d10a8272cb61118579",
       "tag": "v2.4.0"
     },
+    {
+      "actionLink": "actions/checkout",
+      "actionVersion": "2541b1294d2704b0964813337f33b291d3f8596b",
+      "tag": "v3.0.2"
```

Comment:
Nothing particularly significant in this version, but it seemed worthwhile to have the latest version.